Magitek/Logic/Reaper/Normal/AoE.cs

```diff
@@ -28,7 +28,7 @@ public static async Task<bool> WhorlofDeath()
                 return false;
 
             if (!Combat.Enemies.Any(x => (!x.HasMyAura(Auras.DeathsDesign) || (x.HasMyAura(Auras.DeathsDesign) && !x.HasAura(Auras.DeathsDesign, true, Spells.Slice.AdjustedCooldown.Milliseconds)))
-                                         && x.Distance(Core.Me) <= 5 + x.CombatReach))

```

Comment:
These "radius" abilities on reaper appear to not extend to the combat reach of the unit, at least on a striking dummy.